python/compare_data_files.py

```diff
@@ -0,0 +1,157 @@
+# Compare two data files
+import pandas as pd
+
+from python.utils import load_data_file
+
+DATA_DIRECTORY = ".data/compare_jsons"
+OLD_DATA_PATH = f"{DATA_DIRECTORY}/pff_metadata_2020.json"
+NEW_DATA_PATH = f"{DATA_DIRECTORY}/pff_metadata_2021.json"
+OUTPUT_FILE_PATH = f"{DATA_DIRECTORY}/compare_result.csv"
+
+MAX_COMPARISON_ROWS = 100
+NORMALIZE_VALUES = True
+
+INDEX_COLUMN = "pff_variable"
+
+
+def compare_data(old_data: pd.DataFrame, new_data: pd.DataFrame) -> None:
+    print("PREVIEW OLD DATA")
+    print(old_data.head().to_markdown())
+    old_data.info()
+    print("PREVIEW NEW DATA")
+    new_data.info()
+    print(new_data.head().to_markdown())
```

Comment:
I did not know about `to_markdown` for a df, thank you for bringing this knowledge to me